src/components/WatchList/watchList.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { ComponentStory } from '@storybook/react';
```

Comment:
![Screenshot 2022-05-25 at 18-09-48 Pangolin _ WatchList - Default ⋅ Storybook](https://user-images.githubusercontent.com/6604146/170264046-ad5a5137-5ebb-47eb-ba10-6bf7e3bbb543.png)

Storybook chart and token price not showing  
also make fixed width div and put the component inside

Reply:
still here price not showing